packages/core-data/src/entity-types/base.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * Internal dependencies
+ */
+import type { Context, OmitNevers } from './helpers';
+import type { BaseEntityRecords as _BaseEntityRecords } from './base-entity-records';
+
+type TemplatePartArea = {
+	area: string;
+	label: string;
+	icon: string;
+	description: string;
+};
+
+type TemplateType = {
+	title: string;
+	description: string;
+	slug: string;
+};
+
+declare module './base-entity-records' {
+	export namespace BaseEntityRecords {
+		/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
+		export interface Base< C extends Context > {
+			/**
+			 * Site description.
+			 */
+			description: string;
+
+			/**
+			 * GMT offset for the site.
+			 */
+			gmt_offset: string | number;
```

Comment:
Won't it always be a `string` because it comes from a value of the the `wp_options` table?

Reply:
Yeah, that is correct. I don't remember why I set it as a number in the first place.